test/cowboy_SUITE.erl

```diff
@@ -283,9 +284,7 @@ stop_cowboy() ->
 
 execute_request(Host, Path, Method, Headers, Body) ->
     {ok, Pid} = fusco:start_link(Host, []),
-    Response = fusco:request(Pid, Path, Method, Headers, Body, 5000),
-    fusco:disconnect(Pid),
```

Comment:
We can still assure a proper cleanup by returning the Pid and closing the connection once everything has been verified.

OR

We must have a comment here that explains why there is no disconnect. In 1 or 2 years we might completely forget why there is no disconnection here and try to "fix" it.

Reply:
I will add comment